src/data/BackendInterface.hpp

```diff
@@ -407,6 +420,17 @@ class BackendInterface {
     virtual std::optional<Blob>
     doFetchLedgerObject(ripple::uint256 const& key, std::uint32_t sequence, boost::asio::yield_context yield) const = 0;
 
+    /**
+     * @brief The database-specific implementation for fetching a ledger object sequence.
+     *
+     * @param key The key to fetch for
+     * @param sequence The ledger sequence to fetch for
+     * @param yield The coroutine context
+     * @return The object as a sequence on success; nullopt otherwise
```

Comment:
Confusing a bit: i think this returns the sequence if available, not an object as a sequence